Getting started: finding kata (#61)
codewars/docs: content/getting-started/finding-kata.md

# Finding Kata

Your account is configured and you are ready for your next challenge. Now you will learn how to find a task to train on.

## Codewars trainer

The easiest way of getting started is to use our trainer on the dashboard to pick your next kata.

> ![training routines](./img/finding-kata_01_training-routines.png)

Reviewer: Can you take new screenshots so it shows the current UI? I'll update the popup too.

Author: I updated screenshots but it turns out I have no proper image editing software installed on my office laptop, so I am not really satisfied with the result.
Any help here would be welcome.

Reviewer: We can improve it later, but here's my take:
![sidebar](https://user-images.githubusercontent.com/639336/87838157-e1398600-c84a-11ea-8091-0ce93ff4a685.png)
![training-routines](https://user-images.githubusercontent.com/639336/87837996-56f12200-c84a-11ea-95b9-ecc8089d8dee.png)

We might need to retake the screenshot and make sure it's not too wide. I'll look too short on our docs.

# Finding Kata

Your account is configured and you are ready for your next challenge. Now you will learn how to find a task to train on.

> ## Codewars trainer

Reviewer 2: ```suggestion
## Codewars training suggestions
```

Reviewer: It's called "Personal Trainer" internally. Like those in gyms.

> ## Codewars trainer

Reviewer: ```suggestion
## Personal Trainer
```

> The easiest way of getting started is to use our trainer on the dashboard to pick your next kata.

Reviewer 2: ```suggestion
The easiest way of getting started is to use the suggestions box on the dashboard to pick your next kata.
```

see #74

> The easiest way of getting started is to use our trainer on the dashboard to pick your next kata.

Reviewer: ```suggestion
The easiest way of getting started is to use the personal trainer on the dashboard to pick your next kata.
```

![training routines](./img/finding-kata_01_training-routines.png)

> The trainer allows you to pick your language, select your focus, preview the challenge, skip to another challenge and to start training.

Reviewer 2: ```suggestion
The suggestion area allows you to preview the suggested challenge, pick your language, select your focus, then to start training or skip to another challenge.
```
to keep the order with the picture (just the preview is a bit weird to place... let's try it in first position)

> The trainer allows you to pick your language, select your focus, preview the challenge, skip to another challenge and to start training.

Reviewer: ```suggestion
The personal trainer suggests you a kata based on the selected language and focus area. You can either train on the suggested kata or skip to see the next suggestion.
```

For newer developers, we recommend that you pick the "Fundamentals" focus track, which will randomly select easier kata for you to train on.

For experienced developers looking for a challenge, you can select the "Rank Up" focus track, which will cause you to begin with easy to medium challenges and quickly advance to harder ones.

If, for some reason, you do not like suggested kata, you can skip on it without any serious consequences. It will not be suggested to you by trainer anymore, but you will be always able to get back to it and solve it, for example by searching through kata library (see below).

## Kata search page

> For those really looking for a hard challenge, we recommend you go directly to the full list of kata where you can select from over 6000 kata to train on. Remember, 1 kyū is the hardest level a kata can be.

Reviewer 2: ```suggestion
For those really looking for a hard challenge or for a specific kind of rank/task/..., we recommend you go directly to the full list of kata where you can select from over 8000 kata to train on. Remember, 8 kyū is the easiest level a kata can be.
```

* #74 about passive/possessive form again (I won't mark it in the next suggestions
* changed 1 kyu to 8 kyu, this way, still up to date if 1 dan katas are created one day... x)

![sidebar](./img/finding-kata_02_sidebar.png)

> Details of kata search page are described in [UI documentation (TODO: insert link to documentation of kata search panel)](), but here are some hints useful for beginners:

Reviewer 2: ```suggestion
Details of kata search page are described in [UI documentation (TODO: insert link to documentation of kata search panel)](), but here are some useful hints for beginners:
```
